monad-eth-testutil/examples/txgen/config.rs

```diff
@@ -414,6 +473,14 @@ fn default_tx_type() -> TxType {
     TxType::ERC20
 }
 
+fn default_total_authorizations() -> usize {
```

Comment:
nit: using functions for default values is non-idiomatic. Prefer inline values if it's only used once or a named const when used multiple times (i.e. `DEFAULT_AUTHORIZATIONS_PER_TX: usize = 1;`